sections/semantics-tabular-data.include

```diff
@@ -2950,4 +2954,77 @@
     </tbody>
   </table>
 
+  <hr>
+
+  The following shows how one might mark up a coffee order that includes the team,
+  types of coffee, how many cups, and if sugar is needed to fulfill the order:
+
+  <xmp highlight="html">
+    <table>
+      <caption>
+        Daily coffee order for team meetings
+      </caption>
+      <thead>
+        <tr>
+          <th id="t1">Team</th>
+          <th id="t3" abbr="Types" colspan="2">Types of Coffee</th>
+          <th id="t2" colspan="2">Cups</th>
+          <th id="t4">Sugar Packets?</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td headers="t1">Design</td>
+          <td headers="t3" id="c1">House Blend</td>
+          <td headers="t3" id="c2">French Roast</td>
+          <td headers="t2 c1">8</td>
+          <td headers="t2 c2">3</td>
+          <td headers="t4">Yes</td>
+        </tr>
+        <tr>
+          <td headers="t1">Development</td>
+          <td headers="t3" id="c3">Cold Brew</td>
+          <td headers="t3" id="c4">Espresso</td>
+          <td headers="t2 c3">6</td>
+          <td headers="t2 c4">5</td>
+          <td headers="t4">No</td>
+        </tr>
+      <tbody>
+    </table>
+  </xmp>
+
+  This table could look like this:
```

Comment:
that wording makes sense when the example is illustrated with a picture of what it might produce. I think it makes more sense to say something like

> this example produces the following table (in your browser)

Reply:
@chaals i was following how the other table examples were noted.  I agree with you though, but I think I should go in and change all instances of this.

Reply:
Though, there is CSS that is applied to these tables, so *could* isn't completely inappropriate here, as by default these tables won't look like the output in the spec.

Reply:
@chaals I've updated the examples per your suggestion, but also added a bit of introductory text to the section so we only have to mention "in your browser" once, as well as allude to the fact there's some CSS applied to the tables for styling.